frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -346,13 +346,13 @@ describe("scenarios > visualizations > pivot tables", () => {
 
     // sort descending
     cy.get(".Icon-arrow_down").click();
-    cy.findByText("300 – 302.5");
-    cy.findByText("2.5 – 5").should("not.exist");
+    cy.findByText("158 – 160");
+    cy.findByText("8 – 10").should("not.exist");
```

Comment:
@paulrosenzweig there are now only round values (integers) for this range. Does that seem ok?

Reply:
👍 yup! That should be fine.